### PR TITLE
[PD 3114] remove beta feedback link

### PR DESCRIPTION
**Pull request by mwarnsleyDE**
## What's this PR do?
This PR removes the Leave Feedback link from the Analytics sidebar

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the Analytics App

## How should this be manually tested?
Pull down the repository. When you go into the Analytics application you will see that the Leave Feedback has been removed from the left hand sidebar

### CODE REVIEW

File: gulp/src/sass/analytics.scss [hunk 1]
```diff
@@ -9,22 +9,6 @@ $primary-red: #A95051;
 $primary-green: #63AB62;
 
 /* NEED TO DELETE WHEN FEEDBACK IS GONE*/
```
Comment (koonsies) [inline gulp/src/sass/analytics.scss]: Should this comment also go?

**Comment by mwarnsleyDE** [comment]
@koonsies yes, just removed that

**Comment by mwarnsleyDE** [comment]
Need to check with Jason before merging

**Comment by tdphillips** [comment]
There's a feedback link in `gulp/src/analytics/components/SideBar/MobileDimensions.js` as well.

**Pull request merged**